docs/pages/core-concepts.mdx

```diff
@@ -38,16 +38,22 @@ Read our guides to how [authorization](./architecture/authorization.mdx) and
 
 ### Teleport Proxy Service
 
-The **Teleport Proxy Service** allows for secure access to resources in a your
-infrastructure from the public internet without the need for a VPN. 
+The **Teleport Proxy Service** allows for secure access to resources in your
+infrastructure from the public internet without the need for a VPN.  The
+Proxy Service provides connectivity for users, machines and resources
+within the cluster.
 
-It establishes reverse tunnels to the **Teleport Auth Service** and **Teleport
+It establishes reverse tunnels to the **Teleport Auth Service** for **Teleport
 Services**, which can run in private networks. This means that, in the Proxy
 Service's minimal configuration, you can expose only port `443` to the internet
 and run the rest of your infrastructure in private networks.
 
-You can also configure clients to bypass Proxy Service instances and connect to
-resources with Teleport-issued certificates directly. 
+You can run multiple Proxy Service instances in a cluster for high availability.
+
+The Proxy Service, and all Teleport Services, can run on the public internet,
+isolated networks or even air-gapped infrastructure. You can also configure 
+clients to bypass Proxy Service instances and connect to resources with 
+Teleport-issued certificates directly. 
```

Comment:
I'd like to keep the Core Concepts guide more of a conceptual overview than a detailed architectural breakdown. 

Are prospective users reading this guide and getting turned off by the possibility that the Proxy Service only runs on the public internet? If so, we might be able to work this information into this section more smoothly to keep it concise.

Reply:
👍 I think we can streamline this section a bit since the current wording says the Proxy Service enables public internet connections in the first paragraph. How about:

```markdown
The Teleport Proxy Service allows for secure access to resources in your
infrastructure without the need for a VPN, and provides connectivity
 for users, machines and resources within the cluster.

The Proxy Service establishes reverse tunnels to the Teleport Auth Service for Teleport services, which can run in private networks. This means that, in the Proxy Service's minimal configuration, you can expose only port 443 to the internet and run the rest of your infrastructure in private networks.

Like all Teleport services, the Proxy Service can run on the public internet, isolated networks or air-gapped infrastructure. You can also configure clients to bypass Proxy Service instances and connect to resources with Teleport-issued certificates directly.

You can run multiple Proxy Service instances in a cluster for high availability.
```